### DISCUSSION TITLE
V5 Carousel : using carousel with grid columns issue!

### DISCUSSION THREAD

**Discussion by devhoussam** [twbs/bootstrap#33566] [Q&A]
I'm trying to use v5 carousel with grid columns but I get some issue h-100 issues here is an example : 

![Image](https://i.imgur.com/L2Ne7mw.png)

https://codepen.io/themes4all/pen/zYNdWMY

**Comment by mdo** [comment]
This isn't a bug, this is a how-to question. Converting to a Discussion.

**Reply by wnunezc** [reply]
I think the problem is the content ... or the size of the image.

the real question is what size should it be?

If we know that it is a carousel, it is assumed that the exto must be of an adequate size so that all the slides also have the same size and everything is homogeneous, including the sizes of the images (height) ...

then can the carousel adapt the size of the font dynamically to the height of the image so that it does not overshoot and avoid overflow?
